Modules/Forum/classes/class.ilForumTopic.php

```diff
@@ -285,15 +286,18 @@ public function countActivePosts(bool $ignoreRoot = false) : int
 
     public function getFirstPostNode(bool $isModerator = false, bool $preventImplicitRead = false) : ilForumPost
     {
+        $this->db->setLimit(1);
         $res = $this->db->queryF(
             '
 			SELECT *
 			FROM frm_posts 
 			INNER JOIN frm_posts_tree ON pos_fk = pos_pk
-			WHERE parent_pos = %s
-			AND thr_fk = %s',
+			WHERE parent_pos != %s
```

Comment:
IMO This is wrong in this case, I checked the usages. We should rename this method to `getPostRootNode`. The `depths` MUST be 1 and the `parent_pos` MUST equal `0`.